### PR TITLE
personal working branch to personal main branch PR

### PR DESCRIPTION
**Pull request by ionutchinan**
First batch of tests.

### CODE REVIEW

File: cypress/integration/users.spec.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    })
+
+    it('should correctly display an empty table when entering invalid "Search By" input', () => {
+        cy.get('@searchInputBox').type('ab#1;')
+        cy.get('@rowName').should('not.exist')
+        cy.get('@searchInputBox').clear()
+    })
+
+    it('should correctly load stubbed /users API call response in the table', () => {
+        const userArray = [
```
Comment (vladMocan) [inline cypress/integration/users.spec.js:60]: you already have this code in fixtures please use if from there,

File: cypress/integration/landingPage.spec.js [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+    beforeEach(() => {
+        cy.get('[data-testid="user-list"]').as('userListNavButton')
+    })
+
+    it('should redirect to /users', () => {
+        cy.location('pathname').should('equal', '/users')
+    })
+
+    it('should have title "QA"', () => {
+        cy.title().should('equal', 'QA')
```
Comment (vladMocan) [inline cypress/integration/landingPage.spec.js:17]: Try to store strings intro constant file.

File: cypress/integration/users.spec.js (hunk 1 above)
Comment (ionutchinan) [reply cypress/integration/users.spec.js]: fixed!

File: cypress/integration/landingPage.spec.js (hunk 2 above)
Comment (ionutchinan) [reply cypress/integration/landingPage.spec.js]: done!

**Pull request merged**